src/pages/Member/AdminUpdateClass/DeleteClassModal.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { deleteClass } from '../../../api/apiPlanClass';
+
+import { Modal } from '../../../components/Modal/Modal';
+import { Button, PrimaryButton } from '../../../components/Button';
+import { ErrorMessageInterface } from '../../../types';
+import { StyledErrorMessage } from '../../../components/ErrorMessage';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+`;
+
+const Text = styled.h4`
+  display: flex;
+  gap: 1em;
+  align-items: center;
+
+  span {
+    font-size: 1.5rem;
+  }
+`;
+
+interface deleteClassProps {
+  className: string;
+  classID: string;
+  isOpen: boolean;
+  toggle(): void;
+  handleClose(): void;
+}
+
+export const DeleteClassModal = ({
+  className,
+  classID,
+  isOpen,
+  toggle,
+  handleClose,
+}: deleteClassProps) => {
+  const [errorMessages, setErrorMessages] = useState<ErrorMessageInterface[]>(
+    []
+  );
+
+  const handleSubmit = async () => {
+    try {
+      await deleteClass(classID);
+      handleClose();
```

Comment:
If what this function (handleClose()) does is to re-fetch the classes could we just pass the fetchClass function instead? I think the naming is a little bit confusing. 🤔